README.md

```diff
@@ -1,59 +1,29 @@
-
-</div>
-
-BlockScout provides a comprehensive, easy-to-use interface for users to view, confirm, and inspect transactions on EVM (Ethereum Virtual Machine) blockchains. This includes the POA Network, xDai Chain, Ethereum Classic and other **Ethereum testnets, private networks and sidechains**.
-
-See our [project documentation](https://docs.blockscout.com/) for detailed information and setup instructions.
-
-Visit the [POA BlockScout forum](https://forum.poa.network/c/blockscout) for FAQs, troubleshooting, and other BlockScout related items. You can also post and answer questions here.
-
-You can also access the dev chatroom on our [Gitter Channel](https://gitter.im/poanetwork/blockscout).
-
-## About BlockScout
-
-BlockScout is an Elixir application that allows users to search transactions, view accounts and balances, and verify smart contracts on the Ethereum network including all forks and sidechains.
-
-Currently available full-featured block explorers (Etherscan, Etherchain, Blockchair) are closed systems which are not independently verifiable.  As Ethereum sidechains continue to proliferate in both private and public settings, transparent, open-source tools are needed to analyze and validate transactions.
-
-## Supported Projects
-
-BlockScout supports a number of projects. Hosted instances include POA Network, xDai Chain, Ethereum Classic, Sokol & Kovan testnets, and other EVM chains. 
-
-- [List of hosted mainnets, testnets, and additional chains using BlockScout](https://docs.blockscout.com/for-projects/supported-projects)
-- [Hosted instance versions](https://docs.blockscout.com/about/use-cases/hosted-blockscout)
-
-
-## Getting Started
-
-See the [project documentation](https://docs.blockscout.com/) for instructions:
-- [Requirements](https://docs.blockscout.com/for-developers/information-and-settings/requirements)
-- [Ansible deployment](https://docs.blockscout.com/for-developers/ansible-deployment)
-- [Manual deployment](https://docs.blockscout.com/for-developers/manual-deployment)
-- [ENV variables](https://docs.blockscout.com/for-developers/information-and-settings/env-variables)
-- [Configuration options](https://docs.blockscout.com/for-developers/configuration-options)
-
-
-## Acknowledgements
-
-We would like to thank the [EthPrize foundation](http://ethprize.io/) for their funding support.
+<p align="center">
+  Blockchain Explorer for inspecting and analyzing chains of the Trustlines Foundation
+</p>
 
-## Contributing
+This is a fork of the [origin Blockscout
+project](https://github.com/poanetwork/blockscout). It is an adjusted version
+for the purpose of the [Trustlines
+Blockchain](https://explore.tlbc.trustlines.foundation/) and the
+[Laika](https://explore.laika.trustlines.foundation/) test network. Such include
+adoptions of the view and layout focusing on the needs of the Trustlines
+Foundation.
```

Comment:
"Such include
adoptions of the view and layout focusing on the needs of the Trustlines
Foundation."

Doesn't parse in my brain. I'd just delete it. Otherwise LGTM

Reply:
Haha okay I did so. :smile: